src/__test__/detailsPage.test.js

```diff
@@ -0,0 +1,23 @@
+import renderer from 'react-test-renderer';
+import { cleanup } from '@testing-library/react';
+import { Provider } from 'react-redux';
+import { BrowserRouter, Route, Routes } from 'react-router-dom';
+import store from '../redux/store';
+import DetailsPage from '../pages/DetailsPage';
+
+afterEach(cleanup);
+test('Check if Details page is rendering correctly', () => {
+  const detailsPage = renderer
+    .create(
+      <Provider store={store}>
+        <BrowserRouter>
+          <Routes>
+            <Route path="search/details" element={<DetailsPage />} />
+          </Routes>
+          ,
+        </BrowserRouter>
+      </Provider>,
+    )
+    .toJSON();
+  expect(detailsPage).toMatchSnapshot();
+});
```

Comment:
- [ ] Following the project requirements, you should use also React Testing Library to simulate user interaction. Check out this [link ](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/articles/testing_react_with_react_testing_library.md)to learn how to do that.

Reply:
I thank you @medaminedev66. It was a skip in activity. Thanks for pointing it out. I will fix that immediately.

Reply:
Issue fixed, and test file is updated. Thanks @medaminedev66